src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcDumpDumper.cs

```diff
@@ -86,7 +98,7 @@ public void AttachToTargetProcess(int processId, string outputDirectory, DumpTyp
             throw new InvalidOperationException("Procdump crash dump file must end with .dmp extension.");
         }
 
-        if (!TryGetProcDumpExecutable(processId, out var procDumpPath))
+        if (!new ProcDumpExecutableHelper(_processHelper, _fileHelper, _environment, _environmentVariableHelper).TryGetProcDumpExecutable(processId, out var procDumpPath))
```

Comment:
Might be better to new this up in ctor and reuse it.

Reply:
done